packages/frontend/.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_API_URL=http://localhost:8080
```

Comment:
See: https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables

Please rename this file to `.env.development`.